### DISCUSSION TITLE
Welcome to hugo-theme-dream Discussions!

### DISCUSSION THREAD

**Discussion by g1eny0ung** [g1eny0ung/hugo-theme-dream#161] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
This is awesome. I must say, your hugo theme has way better documentation than the rest of the themes out there.

**Reply by g1eny0ung** [reply]
Thx! I'm glad you like this theme.
